src/components/Page.vue

```diff
@@ -1,6 +1,7 @@
 <template>
-  <div
-    :class="[fill, { desktop: !$globals.IS_MOBILE_DEVICE }]"
+  <component
+    :is="renderAs"
```

Comment:
may be clearer to have it inline:
```suggestion
    :is="modal ? AeModal : 'div'"
```